frontend/plugins/frontline_ui/src/widgets/automations/modules/facebook/components/action/components/FacebookCardsMessage.tsx

```
@@ -0,0 +1,143 @@
+                  + add page
+                </Button>
+              </div>
+            </div>
+            <Tabs>
+              <div className="overflow-x-auto p-2">
+                <Tabs.List
+                  size="sm"
+                  defaultValue={cards?.length === 1 ? '1' : undefined}
+                >
+                  {cards.map((_, index) => (
+                    <Tabs.Trigger size="sm" value={String(index)}>{`${
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)